view/frontend/web/js/pargo.js

```diff
@@ -8,668 +8,626 @@
-      {
-          var pargoStreet1 = point.data.suburb;
-          var pargoStreet2 = "";
-      } else {
-          var pargoStreet1 = point.data.address2;
-          var pargoStreet2 = point.data.suburb;
-      }
-
-    $(".form-shipping-address")
-      .find("select[name=country_id]")
-      .val("ZA")
-      .change();
-    $(".form-shipping-address").find(nameSelector).val(name).change();
-    $(".form-shipping-address").find(lastnameSelector).val(lastname).change();
-    $(".form-shipping-address")
-      .find("input[name=company]")
-      .val(point.data.storeName + "-" + point.data.pargoPointCode)
-      .change();
-    $(".form-shipping-address")
-      .find('input[name="street[0]"]')
-      .val(point.data.address1)
-      .change();
-    $(".form-shipping-address")
-      .find('input[name="street[1]"]')
-      .val(pargoStreet1)
-      .change();
-    $(".form-shipping-address")
-      .find('input[name="street[2]"]')
-      .val(pargoStreet2)
-      .change();
-    $(".form-shipping-address")
-      .find("input[name=city]")
-      .val(point.data.city)
-      .change();
-    $(".form-shipping-address")
-      .find("input[name=region]")
-      .val(point.data.province)
-      .change();
-    $(".form-shipping-address")
-      .find("input[name=postcode]")
-      .val(point.data.postalcode)
-      .change();
-    $(".form-shipping-address").find(telephoneSelector).val(telephone).change();
-
-    pargoAlternativeDisplay();
-    /* Custom updates */
-    jQuery("div#checkout-step-shipping_method").css({
-      opacity: "0.5",
-      "pointer-events": "none",
-    });
-
-    setTimeout(function () {
-      jQuery(".radio").each(function () {
-
-        var value = jQuery(this).val();
+    //pargo can be loaded just set the shipping rate
+    if (
+      pargoPointState === true &&
+      checkout.getSelectedShippingRate() !==
+        "pargo_customshipping_pargo_customshipping"
+    ) {
+      loadPargoInformation = true;
+      pargoPointState = true;
+      checkout.setSelectedShippingRate(
+        "pargo_customshipping_pargo_customshipping"
+      );
+    }
+    */
+    // * changes start
+    if(pargoPointState===true){
+        loadPargoInformation = true;
+        //checkout.setSelectedShippingRate("pargo_customshipping_pargo_customshipping");
+    }
+    // * changes end
 
-        if (value == "pargo_customshipping_pargo_customshipping") {
+    // iFrame close event listener
+    if (window.addEventListener) {
+        window.addEventListener("message", setPargoPointInformation, false);
+    } else {
+        window.attachEvent("onmessage", setPargoPointInformation);
+    }
 
-        var pargoPoint = JSON.parse(localStorage.getItem("pargoPoint"));
-/*
-        var pargoPointTelephone = pargoPoint.phoneNumber.toString();
-        if(pargoPointTelephone==="")
-            pargoPointTelephone = "[]";
+    function setPargoPointInformation(point) {
 
-*/
+        console.log("Pargo: setPargoPointInformation");
```

Comment:
Remove console log

Reply:
This is still required for debugging on the staging site, will remove when committing to main / master.